src/conv/problem_description.cpp

```diff
@@ -127,6 +139,16 @@ void ProblemDescription::BuildConfKey(std::string& conf_key) const
         ss << 'x' << GetOutLayout();
     }
     ss << 'x' << EncodeDataTypesForKey(GetInDataType(), GetWeightsDataType(), GetOutDataType());
+    if(GetInCastType() || GetWeightsCastType() || GetOutCastType())
```

Comment:
Similar to previous comment

Reply:
reduced cast format to 
x<ciF16><cwF16><coF16>
where each <> is optional, labels remain in case not all 3 aren't present
the phrase with leading x isn't included if there are no casts

Reply:
[resolved]